examples/plot_decoding_tutorial.py

```diff
@@ -208,14 +215,26 @@
 # Cross-validation with the decoder
 # ...................................
 #
-# The decoder implements a cross-validation loop by default, it also returns
-# an array of shape (cross-validation parameters, n_folds)..
-decoder = Decoder(estimator='svc', mask=mask_filename, standardize=True,
-                  cv=cv)
+# The decoder also implements a cross-validation loop by default and returns
+# an array of shape (cross-validation parameters, `n_folds`). We can use
+# accuracy score to measure its performance. This is done by defining
+# `accuracy` as the `scoring` parameter.
+n_folds = 5
+decoder = Decoder(
+    estimator='svc', mask=mask_filename,
+    standardize=True, cv=n_folds,
+    scoring='accuracy'
+)
 decoder.fit(fmri_niimgs, conditions)
 
-print(decoder.cv_scores_['face'])
-print(decoder.cv_scores_['cat'])
+kfold_scores = decoder.cv_scores_['face']
+
+for fold in range(n_folds):
+    print(
+        "CV Fold {:01d} | Prediction Accuracy: {:.3f}".format(
+            fold + 1, kfold_scores[fold]))
+avg_accuracy = sum(kfold_scores) * 100 / n_folds
+print("Decoder average cross-validation accuracy : %f%%" % avg_accuracy)
```

Comment:
I wonder whether we should not use sklearn.model_selection.cross_val_score here? We are giving bad habit to users.

Reply:
I agree but the idea is to showcase the 'cv_scores_' parameter of 'Decoder' ?

Reply:
The idea is indeed to demonstrate Decoder object has the attrbute `cv_scores_`. But I will give some more lines to clarify that in other cases users should use other cross-validation scheme that Decoder object does not support.

Reply:
But those scores are not nested-crossvalidation scores. I wonder how much we want to showcase them in a tutorial.

Reply:
Oh, I see these are the parameters learning scores. Indeed, we should _not_ display them.